practiseCoder/views.py

```diff
@@ -5,7 +5,7 @@
 
 def index(request, user):
 	global jsonData
-	with open('probelms.txt', 'r') as file:
+	with open('practiseCoder/probelms.txt', 'r') as file:
```

Comment:
Better idea, check if probelms.txt exist in the location '.' if it doesn't, then execute helper.py to generate the file. Also move helper.py to {root} (the location where manage.py is present)